gammapy/datasets/spectrum.py

```diff
@@ -429,3 +431,163 @@ def to_spectrum_dataset(self, name=None):
             SpectrumDataset with Cash statistic.
         """
         return self.to_map_dataset(name=name).to_spectrum_dataset(on_region=None)
+
+
+class SpectrumDatasetChi2(SpectrumDataset):
+    def __init__(self, *, counts, sigma, **kwargs):
+        """
+        Parameters
+        ----------
+        counts : `~gammapy.maps.RegionNDMap`
+            On counts.
+        sigma : `~gammapy.maps.RegionNDMap`, optional
```

Comment:
Maybe `counts_errors` or `counts_stat_errors` is more explicit.

Reply:
* I am not sure if it is of used, right now, beyond X-ray/optical astronomy. 
* you mean like a systematic term as in UVOT? I am not sure IXPE provides it. But sure, one could think of adding an additional component. But I thought we agreed that systematics in gammapy are better handled within the model using priors. 
* Agree, sqrt(npred) can be a good alternative.

Reply:
Is it not possible to handle the case of having error_plus and error_minus?

Reply:
The OGIP format allows that, by memory... But maybe one could do that in the second step